samples/bluetooth/broadcast_audio_source/boards/nrf52833dk_nrf52833.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_MAIN_STACK_SIZE=4096
+
+# Use USB Audio as input
+CONFIG_USE_USB_AUDIO_INPUT=y
+CONFIG_USB_DEVICE_PRODUCT="Zephyr Broadcast Source"
```

Comment:
We could consider making this the default

Reply:
Not all SoCs support USB, lets keep the USB Kconfig in board configs for now.

Reply:
Fair